core/protos/protos/network.proto

```diff
@@ -78,10 +88,11 @@ message PeerMessage {
         BlockHeader block_header_announce = 6;
         bytes block_request = 7;
         Block block = 8;
-        BlockApproval block_approval = 9;
-        SignedTransaction transaction = 10;
-        StateRequest state_request = 11;
-        StateResponse state_response = 12;
-        AnnounceAccount announce_account = 13;
+        SignedTransaction transaction = 9;
```

Comment:
two things:
 - why did you remove BlockApproval specifically?
 - also generally, it's really bad idea to change ids for proto fields. It's better to have empty ids if something gets removed.

Reply:
I mean were Tx and BlockApproval is only thing that requires account_id routing?

Reply:
1. When we introduce chunks the list might grow, but as now, those two are the only types of messages requiring account id.

2. I'm not familiar with protos (haven't used them before). I guess this is necessary to be compatible among different versions.